### ISSUE TITLE
Add filter by Tags  to the UI

### ISSUE THREAD

**Issue by golliholzland** [zenstruck/messenger-monitor-bundle#34]
A great feature would be to add a filter with all possible options for tags, or autocomplete in dashboard Home or history page ![image](https://github.com/zenstruck/messenger-monitor-bundle/assets/132348430/4a65b942-4b8c-4e92-ab47-6e194502f17e)

**Comment by Chris53897** [comment]
UX-Autocomplete could be a good solution for this.

**Comment by kbond** [comment]
Good idea! I'd want it to work w/o autocomplete (just a free-form input) but would be nice if autocomplete is installed/available, use this. Kind of like I do with the live components.

**Comment by golliholzland** [comment]
we need also status filter
